src/extension/background-script/actions/nostr/getPrivateKey.ts

```diff
@@ -1,9 +1,30 @@
+import { decryptData } from "~/common/lib/crypto";
+import type { MessagePrivateKeyGet } from "~/types";
+
 import state from "../../state";
 
-const getPrivateKey = async () => {
-  const privateKey = state.getState().getNostr().getPrivateKey();
+const getPrivateKey = async (message: MessagePrivateKeyGet) => {
+  const id = message?.args?.id;
+
+  if (!id) {
+    return {
+      data: state.getState().getNostr().privateKey,
+    };
+  }
+
+  const accounts = state.getState().accounts;
+  if (Object.keys(accounts).includes(id)) {
+    const password = state.getState().password as string;
+    const account = accounts[id];
```

Comment:
would here `const account = state.getState().getAccount()` work? 
so we don't need to do that here